pyroSAR/tests/test_drivers.py

```diff
@@ -177,3 +177,42 @@ def test_archive(tmpdir, testdata, appveyor):
         with pytest.raises(OSError):
             with pyroSAR.Archive(dbfile) as db:
                 db.import_outdated(testdata['archive_old'])
+
+def test_archive_postgres(tmpdir, testdata, appveyor):
+    id = pyroSAR.identify(testdata['s1'])
+    if not appveyor:
+        db = pyroSAR.Archive('test', postgres=True, port=5432)
+        db.insert(testdata['s1'], verbose=False)
+        assert all(isinstance(x, str) for x in db.get_tablenames())
+        assert all(isinstance(x, str) for x in db.get_colnames())
+        assert db.is_registered(testdata['s1']) is True
+        assert len(db.get_unique_directories()) == 1
+        assert db.select_duplicates() == []
+        assert db.select_duplicates(outname_base='S1A__IW___A_20150222T170750', scene='scene.zip') == []
+        assert len(db.select(mindate='20141001T192312', maxdate='20201001T192312')) == 1
+        assert len(db.select(polarizations=['VV'])) == 1
+        assert len(db.select(vectorobject=id.bbox())) == 1
+        assert len(db.select(sensor='S1A', vectorobject='foo', processdir=str(tmpdir), verbose=True)) == 1
+        assert len(db.select(sensor='S1A', mindate='foo', maxdate='bar', foobar='foobar')) == 1
+        out = db.select(vv=1, acquisition_mode=('IW', 'EW'))
+        assert len(out) == 1
+        assert isinstance(out[0], str)
+        with pytest.raises(IOError):
+            db.filter_scenelist([1])
+        db.close()
+        with pyroSAR.Archive('test', postgres=True, port=5432) as db:
+            assert db.size == (1, 0)
+            shp = os.path.join(str(tmpdir), 'db.shp')
+            db.export2shp(shp)
+        assert Vector(shp).nfeatures == 1
+        db.drop_database()
```

Comment:
`db.drop_database()` is called outside of the above `with` statement scope. Is this the `db` object declared in line 184?  
This is a little confusing. What happens to the object when `db.close()` is called?

Reply:
db.close only shuts down orm session and connection. drop_database should be inside the with statement